python/paddle/tensor/creation.py

```diff
@@ -54,6 +55,166 @@
 ]
 
 
+@dygraph_only
+def to_tensor(data, dtype=None, stop_gradient=True, pin_memory=False):
+    """
+    :api_attr: imperative
+
+    :alias_main: paddle.to_tensor
+	:alias: paddle.tensor.to_tensor, paddle.tensor.creation.to_tensor
+    :old_api: paddle.fluid.dygraph.to_variable
+
+    Constructs a ``paddle.Tensor`` or ``paddle.ComplexTensor`` from ``data`` , 
+    which can be scalar, tuple, list, numpy\.ndarray, paddle\.Tensor, paddle\.ComplexTensor.
+
+    If the ``data`` is already a Tensor, and ``dtype`` does't change, no copy will be performed,
+    otherwise a new Tensor will be returned. Similarly, if the data is an ndarray of 
+    the corresponding dtype and the current place is cpu, no copy will be performed.
+
+    The ``ComplexTensor`` is a unique type of paddle. If x is ``ComplexTensor``, then 
+    ``x.real`` is the real part of ``x`` which is a tensor, and ``x.imag`` is the 
+    imaginary part.
+
+    Args:
+        data(scalar|tuple|list|ndarray|Tensor|ComplexTensor): Initial data for the tensor.
+            Can be a scalar, list, tuple, numpy\.ndarray, paddle\.Tensor, paddle\.ComplexTensor.
+        dtype(str, optional): The desired data type of returned tensor. Can be 'bool' , 'float16' , 
+            'float32' , 'float64' , 'int8' , 'int16' , 'int32' , 'int64' , 'uint8'. And
+            'complex64' , 'complex128' only for ComplexTensor.
+            Default: None, infers data type from data.
+        stop_gradient(bool, optional): Whether to block the gradient propagation of Autograd. Default: True.
+        pin_memory (bool, optional): If True, returned tensor would be allocated in the pinned memory. Works 
```

Comment:
If we need to set the destination place of the created tensor, maybe an argument `place` is better. Because `place` can be `CPUPlace`,  `CUDAPlace`,  and `CUDAPinnedPlace`, and other places we will extend in the future. While, argument `pin_memory` can only specify useing pin_memory or not.

Reply:
Done, place will be set to 'cpu'|'pin_memory'|'cuda' or paddle.CPUPlace|paddle.CUDAPinnedPlace|paddle.CUDAPlace flexibility.